projects/VS2022/examples/shapes_splines_drawing.vcxproj

```diff
@@ -202,7 +202,7 @@
       </PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;PLATFORM_DESKTOP;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WIN32;_CRT_SECURE_NO_WARNINGS;_DEBUG;_CONSOLE;PLATFORM_DESKTOP;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Why is this flag added to this project? Is it required? Is it included in the other example projects?

Reply:
Because this project uses sscanf, and without this there will be a security warning. Any project that uses printf/scanf needs to do this.
the raylib project itself has defines for this. any example that uses the functions directly needs the same.

Reply:
I can't find the usage of `sscanf()` (or `printf()`/`scanf()`) in the example code. But I imagine it's because the inclusion of `<stdlib.h>` for the `NULL` pointer definition.